### ISSUE TITLE
Weekly and Monthly filters for viewing transactions

### ISSUE THREAD

**Issue by jtran8** [ayastreb/money-tracker#58]
While using Money Tracker, I discovered it is not very convenient to view transactions on a weekly or monthly basis. I think that these filters would be more useful in than the currently provided filters as most people think of money in terms of calendar weeks and month as opposed to the last 7 days. 

Currently, to get a weekly or monthly view you must use the custom date filter and manually select the range. Quickly changing the date range isn't possible since using the custom date filter deactivates the button. To change the date range, you must click out to a different filter before selecting custom date again to access the range selector. This can get annoying if you just want to view your transactions month-by-month.

My suggestion for implementation would be to simply bring over the filter view from the Reports page, where there are arrow buttons that allow the user to quickly go forward and backwards in the calendar.

**Comment by Geodebs** [comment]
I agree with this idea. Even just adding a weekly report filter would help tremendously! Anyone capable of pulling this off?
